.gitignore

```diff
@@ -91,3 +91,10 @@ artifacts/
 
 # Python version
 .python-version
+
+# Certificate files
+*.crt
+*.key
+*.cer
+*.pem
+*.pvk
```

Comment:
Why are these added?

Reply:
This is so that test certs that are created in the repo during tests aren't accidentally checked in if they fail to get deleted. I can simplify this to only 2 types instead of 4.

Reply:
There are cer files and pem files checked in with batch and keyvault tests, which they treat as test resource. Better not to exclude these file at the global level. You can add `.gitignore` to iot folder to narrow the scope.

```
23:56 $ find . -name '*.cer'
./src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/tests/data/batchtest.cer
./src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/test-root-cert.cer

23:56 $ find . -name '*.pem'
./env/lib/python3.5/site-packages/certifi/cacert.pem
./env/lib/python3.5/site-packages/certifi/old_root.pem
./env/lib/python3.5/site-packages/certifi/weak.pem
./env/lib/python3.5/site-packages/pip/_vendor/requests/cacert.pem
./src/azure-cli-core/azure/cli/core/tests/sp_cert.pem
./src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/tests/import_pem_encrypted_pwd_123.pem
./src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/tests/import_pem_encrypted_pwd_1234.pem
./src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/tests/import_pem_plain.pem
./src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/tests/mydomain.test.encrypted.pem
./src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/tests/mydomain.test.pem
./src/command_modules/azure-cli-role/azure/cli/command_modules/role/tests/cert.pem
```